packages/cli/src/oclif/commands/validate.js

```diff
@@ -57,8 +58,14 @@ class ValidateCommand extends BaseCommand {
       checkResult = await validateApp(rawDefinition);
     }
 
+    const doneMessage = checkResult.passes
+      ? `Running app checks ... ${checkResult.passes.length} checks passed`
+      : undefined;
+    this.stopSpinner({ message: doneMessage });
+
     const checkIssues = flattenCheckResult(checkResult);
 
+    this.log("\nHere're the issues we found:");
```

Comment:
While I believe `here're` is [technically correct](https://english.stackexchange.com/questions/36532/using-the-contraction-of-are), I'm not sure I've ever read it like that and it comes off as very unusual. I think `Here are` is clearer here.

Reply:
Thanks! I've been writing this for a long time until TIL **here're** (or **there're**) isn't a thing in English. Fixed in a455de9.

Reply:
For sure! `here` is contracted lots of other commonly-accepted ways, but for some reason that's not one of them. English is a mess.